german/Smartphone/Internet/Internet.md

```diff
@@ -0,0 +1,10 @@
+Du bemerkst das das Booten des Smartphones so lange gedauert hat, 
+dass du mittlerweile bis zu den Knien im Wasser stehst.
+Das Ruderboot ist zum Glück aus Holz und geht nicht ganz unter.
+Da du Internetempfang und einen aktzeptablen Akkustand hast, ...
+
+[machst du erstmal ein gut gelauntes Selfie und postest es auf Facebook.](../Rettung/Facebook/Facebook.md)
+
+[schreibst deinem besten Kumpel eine WhatsApp, das er dich an deiner aktuellen GPS Position abholen soll](../Rettung/Kumpel/Kumpel.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.